app/js/controllers/foyer/individuForm.js

```diff
@@ -48,7 +48,7 @@ angular.module('ddsApp').controller('FoyerIndividuFormCtrl', function($scope, in
         role: individuRole
     };
 
-    if (true === ($scope.captureRelationConjoint = !!options.captureRelationConjoint)) {
+    if (true === ($scope.captureRelationConjoint = !! options.captureRelationConjoint)) {
```

Comment:
Just to keep the conversation on truthies/falsies going on, and not asking for changes here as you obviously just fixed whitespace (thanks!), this is an excellent example of added complexity. This could simply be rewritten to:
```js
$scope.captureRelationConjoint = options.captureRelationConjoint;
if ($scope.captureRelationConjoint) {
```